lib/game_teams.rb

```diff
@@ -1,5 +1,5 @@
-require_relative 'stat_tracker'
-
+require_relative './stat_tracker'
```

Comment:
should we use single or double quotes for both lines 1 & 2 to be consistent?

Reply:
fixed.